docs/pages/workflow/customizing.mdx

```diff
@@ -5,21 +5,19 @@ description: Learn how to add custom native code to your Expo project.
+## Create a local module with the Expo Modules API
 
-To use third-party libraries with custom native code and continue with the same developer experience of Expo Go, you can migrate to use [development builds](/develop/development-builds/introduction/).
+By default, Expo projects use [Continuous Native Generation (CNG)](/workflow/continuous-native-generation). This means that projects do not have **android** and **ios** folders containing the native code and configuration used to build the app. Instead, the native projects are generated just before they are built based on your [Expo Config](/workflow/configuration) native modules in **package.json**. This simplifies the management and upgrading of native code, but means you cannot edit the native code directly.
 
-Development builds are your personal version of Expo Go. They include the native runtime that powers your app. You control what is included in that native runtime by adding or removing libraries in the **package.json**. Development builds allow you to continue building your app in JavaScript while taking full advantage of the native libraries available for Expo and React Native projects.
+However, you can still add custom native code to a project using CNG by [creating a local Expo module](modules/get-started/#creating-the-local-expo-module). Local Expo Modules are just like [Expo Modules](/modules/overview/) created by package developers and those used within the Expo SDK itself, but they are not published to npm. Instead, they are created directly inside of your project. This allows you to write custom Swift and Kotlin code that you can interact with from your JavaScript coe, and is included in your development builds and production apps automatically when you build with [EAS Build](/build/introduction) or when you build locally with `npx expo run:[android|iOS]`.
 
-To learn how to start using custom native code in your app by switching from Go to development builds, see [create a development build](/develop/development-builds/create-a-build/).
+## Manage your own native projects
 
-## Generate native projects with prebuild
-
-If you want to take control of Android and iOS native projects and move away from JavaScript-based project, you can generate native directories by running `npx expo prebuild` or `npx expo run:[android|iOS]` (which will run `prebuild` automatically). You can also use development builds in this context &mdash; run `npx expo install expo-dev-client` before `prebuild` or `run`. It is also possible to [add this library](/develop/development-builds/create-a-build/) later.
+If you want to opt out of CNG and instead directly manage the code and configuration inside your Android and iOS native projects, you can generate native directories by running `npx expo prebuild` or `npx expo run:[android|iOS]` (which will run `prebuild` automatically). You can also use development builds in this context &mdash; run `npx expo install expo-dev-client` before `prebuild` or `run`.
```

Comment:
```suggestion
You can opt out of CNG by directly managing code and configuration inside **android** and **ios** directories. To generate these directories, run `npx expo prebuild` or [compile your app locally](/guides/local-app-development/#local-app-compilation) (`npx expo run android` or `npx expo run ios`). You can also create a development build before compiling your app locally. To do so, run `npx expo install expo-dev-client` before `prebuild` or `run`.
```

Reply:
I generally like this, but feel like "by" implies a causal relationship that isn't quite there (there's no native projects to manage until you do something else to opt out of CNG), so accepting and changing a couple of words after :-)